README.md

```diff
@@ -9,19 +9,26 @@ Mount [Microsoft OneDrive][onedrive] storage as [FUSE] filesystem.
 
 ## Installation
 
-*Note: For Nix users, the program is already packaged via Nix Flake in `flake.nix`.*
+_Note: For Nix users, the program is already packaged via Nix Flake in `flake.nix`._
 
 1.  Use your package manager to install these dependencies:
     - pkg-config
     - openssl
     - fuse (libfuse)
+2.  On Debian 12 you may need to also install the following to resolve missing files:
```

Comment:
I use all `1.` to avoid renumbering when inserting or deleting steps, which is exactly what your change is about. Please keep unrelated part untouched.